src/client_disposable.ts

```diff
@@ -9,6 +9,8 @@ import {
 import registerFormatDocument from "./commands/format_document.ts";
 import registerCache from "./commands/cache.ts";
 import registerRenameSymbol from "./commands/rename_symbol.ts";
+import registerFindSymbol from "./commands/find_symbol.ts";
+import registerSymbolSidebarFindSymbol from "./commands/sidebar_find_symbol.ts";
```

Comment:
That you've structured these as two separate commands is actually pretty handy — it means that unlike the TS extension, we can skip alerting the user to open the Deno sidebar when it's invoked from the sidebar and they already have it open.